verification/cbmc_stubs/math.c

```
@@ -1128,7 +1128,7 @@ float fmaxf(float f, float g) { return ((f >= g) || isnan(g)) ? f : g; }
 #endif
 
 // TODO : Should call a __CPROVER_function so that it can be converted to SMT-LIB
-long double fmaxl(long double f, long double g) { return ((f >= g) || isnan(g)) ? f : g; }
+long double avocado_fmaxl(long double f, long double g) { return ((f >= g) || avocado_isnan(g)) ? f : g; }
```

Clang (14.0.6)

[warning] parameter name 'f' is too short, expected at least 3 characters

(readability-identifier-length)